tests/lib/reject.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var expect = require('expect.js'),
+    _ = require('underscore'),
+    reject = require('../../src/lib/reject');
+
+function rejectA(val) {
+   return val === 'a';
+}
+
+function expectEmptyArray(arr) {
+   expect(arr).to.be.an('array');
+   expect(arr).to.be.empty();
+}
+
+describe('reject', function() {
+   it('returns an empty array when given a non-array parameter to filter', function() {
+      expectEmptyArray(reject(undefined, undefined));
+      expectEmptyArray(reject(null, undefined));
+      expectEmptyArray(reject(false, undefined));
```

Comment:
In this case your code actually works regardless of whether the function is falsy or not (as opposed to the `find` test, as noted above). However, you should still be testing for what happens if the array is an invalid value in both scenarios: the function is also invalid, and when the function is valid. That would prevent a bug like the one in `find` from ever creeping into `reject`.